functions/Get-DbaSsisEnvironment.ps1

```diff
@@ -0,0 +1,229 @@
+<#
+.SYNOPSIS
```

Comment:
Actually, since we return one object per variable,
this function should be named Get-DbaSsisEnvironmentVariable.